api/provision/result-fn.ts

```diff
@@ -0,0 +1,30 @@
+import { sqsClient } from "../../utils/aws-sdk/sqs";
+import { SendMessageCommand } from "@aws-sdk/client-sqs";
+import middy from "@middy/core";
+import validator from "@middy/validator";
+import { provisioningResponseSchema } from "../../models/provisioning-jobs";
+import { errorHandlingMiddleware } from "../../utils/middleware/error-handling-middleware";
+
+/**
+ * Result Lambda - Takes Step Fn input and sends to SQS Queue to be consumed
+ * by a scheduled job in Service Now.
+ *
+ * @param stateInput - input to the state task that is processed
+ */
+
+export async function baseHandler(stateInput: Record<string, unknown>): Promise<unknown> {
+  const QUEUE_URL = process.env.PROVISIONING_QUEUE_URL ?? "";
+  console.log("Sending result message to " + QUEUE_URL);
+  await sqsClient.send(
+    new SendMessageCommand({
+      QueueUrl: QUEUE_URL,
+      MessageBody: JSON.stringify(stateInput),
+    })
+  );
+
+  return { ...stateInput, statusCode: 200 };
```

Comment:
I used the `statusCode` earlier as a placeholder on the `sample-fn`, so I don't think it is needed here.

Also, since a message is being sent to a `SQS` queue do we need to return anything from this function?

Reply:
OK I figured as much but didn't want to assume for sure. Have since removed it. And no the return value is going to be ignored but I went ahead and let it return `$`, effectively.